gpii/node_modules/gpii-db-operation/src/DbDataStore.js

```diff
@@ -26,6 +26,10 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 var fluid = fluid || require("infusion");
 var gpii = fluid.registerNamespace("gpii");
 
+if (typeof require !== "undefined") {
```

Comment:
Is there any route left by which this might be included in a browser?

Reply:
It's used in 2-3 places in browser tests, for example, [in the authz server browser tests](https://github.com/the-t-in-rtf/universal/blob/master/gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/test/html/AuthorizationServiceTests.html#L74).

This is another good question for @cindyli or perhaps Simon.  I left the browser stuff alone for now, but if possible I'd prefer converting that as well.  Although in general I'm not a huge fan of mocks, I'd consider that preferable to maintaining gpii-pouch just for that, as long as there are also integration tests that use an actual Couch instance.

IMO this would be a good topic for a future overlap meeting.

Reply:
Yes, as @the-t-in-rtf has discovered, there are 3 browser tests require "DbDataStore.js". They are: auth grant finder, authorization service and preferences service. I believe they were written as browser tests because of browsers' easy-to-debug feature etc, although the preferences service test has been converted to a node test by @sgithens in [his pull request that adds new endpoints for PPT](https://github.com/GPII/universal/pull/634).

I agree I'm also not a huge fan of mocks that are needed for writing these tests as browser tests.

Reply:
The browser tests that used pouch have now been converted, as has the empty key loader.

Reply:
So we can now axe this require sniffing?, as well as the variant infusion load on line 26?